app/frontend/components/pages/dashboard/premiumBanner.tsx

```diff
@@ -0,0 +1,31 @@
+import {h} from 'preact'
+import {connect} from '../../../helpers/connect'
+import actions from '../../../actions'
+
+interface Props {
+  closePremiumBanner: () => void
+}
+
+const PremiumBanner = ({closePremiumBanner}: Props) => {
+  const premiumMessage =
+    'VIP STAKING SERVICES - Do you own more than 3 million ADA? Contact us on  adalite@vacuumlabs.com to join our VIP staking services. We will assign you a dedicated account manager and provide you with priority support.'
```

Comment:
I think 3 million should not be hardcoded here, since its the current PREMIUM_MEMBER_BALANCE_TRESHHOLD says 2.5m

Reply:
Yeah I noticed the half million as well but I have to confirm this with Michael whether this is intentional or a mistake. Tho I dont see a reason to not-hardcode the valuer here, the banner might change in future, and I m not sure how would I convert 2500000000000 to something I could into the text.

Reply:
The difference was intentional indeed.